src/commands/migrate.js

```diff
@@ -8,9 +15,45 @@ const migrate = (context, fromVersion, toVersion, optionalPercent = '100%') => {
     return Promise.resolve();
   }
   optionalPercent = parseInt(optionalPercent, 10);
+
+  let app;
```

Comment:
any reason to scope this outside?

Reply:
Thanks for the suggestion! Fixed in 032f11e with `Promise.all`.